python/example_code/glue/flight_etl_job_script.py

```diff
@@ -0,0 +1,80 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This script is used by the AWS Glue _getting started with crawlers and jobs_ scenario to
+perform extract, transform, load (ETL) operations on sample flight data.
+As part of the demo, it is uploaded to an Amazon Simple Storage Service (Amazon S3)
+bucket so that AWS Glue can access it.
+"""
+
+# pylint: disable=undefined-variable
+
+# snippet-start:[python.example_code.glue.Scenario_GetStartedCrawlersJobs.job_script]
+import sys
+from awsglue.transforms import *
+from awsglue.utils import getResolvedOptions
+from pyspark.context import SparkContext
+from awsglue.context import GlueContext
+from awsglue.job import Job
+
+"""
+These custom arguments must be passed as Arguments to the StartJobRun request.
+    --input_database    The name of a metadata database that is contained in your 
+                        AWS Glue Data Catalog and that contains tables that describe 
+                        the data to be processed.
+    --input_table       The name of a table in the database that describes the data to
+                        be processed.
+    --output_bucket_url An Amazon S3 bucket that receives the transformed output data.  
```

Comment:
--output_bucket_url An S3 bucket that receives the transformed output data.

Reply:
fixed